packages/rocketchat-i18n/i18n/pt.i18n.json

```diff
@@ -14,6 +14,7 @@
   "Accept_with_no_online_agents": "Aceitar sem agentes online",
   "Access_not_authorized": "Acesso não autorizado",
   "Access_Token_URL": "URL do Token de Acesso",
+  "Accessibility": "Acessibilidade",
```

Comment:
You have to add that to en.i18n.json as well, as that is the base language and if it's not there, it will be deleted from others on next merge.